integrationTests/testProcessorNode.go

```diff
@@ -1227,6 +1233,7 @@ func (tpn *TestProcessorNode) initResolvers() {
 }
 
 func (tpn *TestProcessorNode) initInnerProcessors(gasMap map[string]map[string]uint64) {
+	tpn.ArwenChangeLocker = &sync.RWMutex{}
```

Comment:
we should set the ArwenChangeLocker whenever we instantiate the TestProcessorNode struct (this is how was done on the `development` branch)

Reply:
this part is called by all tests - so this instantiation is correct. there will be a lot of conflicts with development. most probably the merge will be done like accept everything from dev.

Reply:
Yup, exactly